README.md

```diff
@@ -426,7 +428,7 @@ For those using Elastic Inference set the image following this format instead:
 ```
 
 Where `REGION` is your AWS region, such as "us-east-1" or "eu-west-1"; `SAGEMAKER_TENSORFLOW_SERVING_VERSION`, `SAGEMAKER_TENSORFLOW_SERVING_EIA_VERSION`, `TENSORFLOW_INFERENCE_VERSION`, `TENSORFLOW_INFERENCE_EIA_VERSION` are one of the supported versions mentioned above; and "gpu" for use on GPU-based instance types like ml.p3.2xlarge, or "cpu" for use on CPU-based instances like `ml.c5.xlarge`.
- 
+
```

Comment:
Formatting is getting messed up down here. I think if you put the directory structure stuff in a code block, you'll avoid this.

Reply:
what part are you referring to? the only directory structure stuff I see is already in a code block:

![Screen Shot 2020-01-17 at 1 14 02 PM](https://user-images.githubusercontent.com/6631887/72646570-51faf680-392b-11ea-90b3-62abfa67025a.png)

Reply:
Maybe it's something else then... but you can see above how everything is bolded... that's usually when some code block gets messed up or special character is triggering the markdown to start rendering like that.

Reply:
ah. upon further inspection, I think it's the use of double underscores in the code directory stuff. It looks okay in the rendering, though.